pkg/services/ngalert/eval/eval.go

```diff
@@ -141,9 +141,6 @@ type ExecutionResults struct {
 	// Results contains the results of all queries, reduce and math expressions
 	Results map[string]data.Frames
 
-	// Errors contains a map of RefIDs that returned an error
-	Errors map[string]error
```

Comment:
This field is never interacted with, outside of `queryDataResponseToExecutionResults` function.

So, there is no need to keep it on an exported type, it only adds clutter. It can just be an intermediate variable in that function.